dev-docs/bidders/afront.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: bidder
+title: Afront
+description: Prebid Afront Bidder Adaptor
+biddercode: afront
+gdpr_supported: true
```

Comment:
please change this to `tcfeu_supported` as discussed in https://github.com/prebid/prebid.github.io/pull/4725 and it needs to be false if you don't have a GVL ID.

Reply:
I went ahead and did this